libsplinter/src/biome/rest_api/actix/key_management.rs

```diff
@@ -38,7 +38,7 @@ pub fn make_key_management_route(
     key_store: Arc<dyn KeyStore<Key>>,
     secret_manager: Arc<dyn SecretManager>,
 ) -> Resource {
-    Resource::build("/biome/users/{user_id}/keys")
+    Resource::build("/biome/users/keys")
```

Comment:
This change should really be in a separate PR, because it is going to require additional work to look at how we are doing routes to see if this is the only one we are going to change.

Reply:
Are you referring to the updates to route or the updates to `authorize_user` as a whole. Those changed were need to write the refresh token endpoint.

Reply:
There are two things that could have been separate from this PR: a) the path change itself, which is a breaking change and completely unrelated to this PR; b) the fix which ignores {user_id} from the URL in favor of using the trusted user id.

I'm fine with it remaining in this PR at this point I guess.